### PR TITLE
PIM-10157: PULLUP 5.0 -> Master

### PR DESCRIPTION
**Pull request by masacc**
**Description (for Contributor and Core Developer)**

EE: https://github.com/akeneo/pim-enterprise-dev/pull/12321

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

File: src/Akeneo/Channel/Bundle/Resources/config/services.yml [hunk 1]
```diff
@@ -143,6 +144,7 @@ services:
             - '@translator'
             - '@Akeneo\Channel\Bundle\Query\Sql\IsChannelUsedInProductProductExportJob'
             - '%pim_catalog.entity.channel.class%'
+            - '@Akeneo\Channel\Bundle\Query\Sql\IsChannelUsedInProductProductExportJob'
```
Comment (mmetayer) [inline src/Akeneo/Channel/Bundle/Resources/config/services.yml:147]:
```suggestion
```

File: src/Akeneo/Channel/Bundle/Resources/config/services.yml [hunk 2]
```diff
@@ -134,6 +134,7 @@ services:
     channel.updater.translatable:
         class: 'Akeneo\Tool\Component\Localization\TranslatableUpdater'
 
+    # @todo @merge master/6.0: accept master arguments order
```
Comment (mmetayer) [inline src/Akeneo/Channel/Bundle/Resources/config/services.yml:137]: to remove

**Review by mmetayer — APPROVED** [review]
GTM once comments are fixed

**Review by BryanS29-akeneo — APPROVED** [review]

**Review by RiffVentura — APPROVED** [review]

**Review by nmarniesse — APPROVED** [review]
GTM after fix

**Pull request merged**